recipes/riverrem/meta.yaml

```diff
@@ -0,0 +1,109 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Ping! Also, please try grayskull to create your recipes. It will make a recipe that is almost "merge ready."